projects/canopy/src/lib/forms/checkbox-group/checkbox-group.component.ts

```diff
@@ -102,20 +102,25 @@ export class LgCheckboxGroupComponent implements ControlValueAccessor {
     this._validationElement = element;
   }
 
-  _value: Array<string> = null;
+  _value: Array<string> = [];
   @Input()
   get value() {
     return this._value;
   }
-  set value(value) {
+  set value(value: Array<string>) {
     this._value = value;
     this.onChange(value);
+
     if (this.checkboxes) {
-      this.checkboxes.forEach((checkbox) => {
-        if (value.includes(checkbox.value.toString())) {
-          checkbox.checked = true;
-        }
-      });
+      if (value.length > 0) {
```

Comment:
I find the double 'if' statement makes this a bit harder to understand, I wasn't sure if this would do the same, or does it not account for certain situations?

```js
if (this.checkboxes) {
  this.checkboxes.forEach((checkbox) => {
    checkbox.checked = value.includes(checkbox.value.toString()) ? true : false;
  })
}
```

Reply:
alternatively invert the if statements as a rule of thumb can reduce nesting

i.e.
```
if (!this.checkboxes) {
  return;
}
```

tools such as resharper (albeit not typescript) are known for promoting this pattern https://www.jetbrains.com/help/resharper/InvertIf.html

Reply:
yeah I like an early return, I think if the first approach doesn't work for some reason this would be a good option. It would be interesting though to see if the first approach doesn't work as it may imply something else odd is happening.

Reply:
Yeah good point, I think it wasn't working before because of some other change. Have returned earlier and simplified the if statements @thatguynamedandy @aaronBery